docs/dev/table/connect.md

```diff
@@ -500,6 +501,39 @@ rowtime:
 The following watermark strategies are supported:
 
 <div class="codetabs" markdown="1">
+<div data-lang="DDL" markdown="1">
+{% highlight sql %}
+-- Sets a watermark strategy for strictly ascending rowtime attributes. Emits a watermark of the
+-- maximum observed timestamp so far. Rows that have a timestamp smaller to the max timestamp
+-- are not late.
+CREATE TABLE MyTable (
+  ts_field TIMESTAMP(3),
+  WATERMARK FOR ts_field AS ts_field
+) WITH (
+  ...
+)
+
+-- Sets a watermark strategy for ascending rowtime attributes. Emits a watermark of the maximum
+-- observed timestamp so far minus 1. Rows that have a timestamp equal to the max timestamp
+-- are not late.
+CREATE TABLE MyTable (
+  ts_field TIMESTAMP(3),
+  WATERMARK FOR ts_field AS ts_field - INTERVAL '0.001' SECOND
```

Comment:
Why not same to Java/Scala?

Reply:
We don't have a shortcut API for `watermarksPeriodicAscending` in SQL DDL.

Reply:
Can we add a example for how to write `watermarksPeriodicAscending`?

Reply:
This is the example.

Reply:
I see